### PR TITLE
test: allow new ubuntu-stable apparmor pattern

### PR DESCRIPTION
**Pull request by jelly**
The new violation has class="file" after operation="open".

### CODE REVIEW

**Comment by jelly** [comment]
This is for https://github.com/cockpit-project/bots/pull/3953

```
audit: type=1400 audit(1665763333.088:67): apparmor="DENIED" operation="open" class="file" profile="virt-aa-helper" name="/var/lib/cockpittest/vm_one/mydisk3" pid=9618 co  mm="virt-aa-helper" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
```

File: test/machineslib.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def setUp(self):
         # FIXME: report downstream; AppArmor noisily denies some operations, but they are not required for us
         self.allow_journal_messages(r'.* type=1400 .* apparmor="DENIED" operation="capable" profile="\S*libvirtd.* capname="sys_rawio".*')
         # AppArmor doesn't like the non-standard path for our storage pools
-        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
+        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" * profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
```
Comment (skobyda) [inline test/machineslib.py:350]: Test so far fails: https://cockpit-logs.us-east-1.linodeobjects.com/pull-835-20221021-105036-d3235509-ubuntu-stable-bots-3953/log.html#26

Shouldn't it be?
```suggestion
        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" .* profile="virt-aa-helper" name="%s.*' % self.vm_tmpdir)
```

**Review by skobyda — CHANGES_REQUESTED** [review]

**Comment by jelly** [comment]
Ok, ubuntu-stable failed:
```
testlib.Error: FAIL: Test completed, but found unexpected journal messages:
audit: type=1400 audit(1666351774.836:98): apparmor="DENIED" operation="open" profile="virt-aa-helper" name="/var/lib/cockpittest/vm_one/mydisk3" pid=8446 comm="virt-aa-helper" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
```

Regex needs to be fixed.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**